BUILD.md

```diff
@@ -1017,6 +1018,32 @@ Android NNAPI Execution Provider can be built using building commands in [Androi
 
 ---
 
+### iOS
+
+#### Prerequisites
+* A Mac computer with latest macOS
+* Xcode
+   * Install Xcode from https://developer.apple.com/xcode/
+* General Info:
+   * iOS Architectures
+
+      | ios_sysroot     | arm64       | x86_64      |
+      |-----------------|:-----------:|:-----------:|
+      | iphoneos        | YES         | NO          |
+      | iphonesimulator | NO          | YES         |
+      armv7, armv7s and i386 architectures are not currently supported.
+   * apple_deploy_target
+
```

Comment:
Link to apple doc describing what the valid values are would be good.

Reply:
Unfortunately, Apple does not have any official document about this